pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -209,7 +209,7 @@ def write_pep0(
             ("Finished PEPs (done, with a stable interface)", finished),
             ("Historical Meta-PEPs and Informational PEPs", historical),
             ("Deferred PEPs (postponed pending further research or updates)", deferred),
-            ("Abandoned, Withdrawn, and Rejected PEPs", dead),
+            ("Abandoned, Withdrawn, Superseded, and Rejected PEPs", dead),
```

Comment:
"Abandoned" isn't a formal status, shall we remove it to keep the header a bit shorter and accurate?

```suggestion
            ("Withdrawn, Superseded, and Rejected PEPs", dead),
```

Reply:
In order:

```suggestion
            ("Rejected, Superseded, and Withdrawn PEPs", dead),
```